opsgenie/resource_opsgenie_service.go

```diff
@@ -85,6 +95,8 @@ func resourceOpsGenieServiceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("name", res.Service.Name)
 	d.Set("team_id", res.Service.TeamId)
 	d.Set("description", res.Service.Description)
+	// Not implemented: it's not possible to change the tags of an existing service.
```

Comment:
Even if we cannot we still need to read it. Is there any problem for this while getting the tags ?

Reply:
The problem is: if you create a service resource with tags `["a", "b"]`, then changes it to `["a", "b", "c"]`, Terraform will produce a diff explaining you it's going to update the service tags with `["a", "b"] => ["a", "b", "c"]`, every time you run Terraform (as it has no way of doing the actual update).

This not implemented read prevent this diff to be displayed, at the expense of making the Terraform user believing everything has been correctly configured.

Both situations are not ideal (I'd prefer Terraform to be silent, with a warning in the documentation, I guess), except having the tags update feature exposed on the API :)

Reply:
there must be an error while reading the tags maybe sdk does not have correct json tags ? 
Otherwise if you change tags in UI you will not be understand while running terraform apply

Reply:
Sorry, I'm not sure to follow our last comment :( :

* You create a service in Terraform with `tags = ["a", "b"]` and the service is created with these tags in Opsgenie
* You change the tags using the UI to `["a", "b", "c"]`
* If I uncomment the code in the PR, Terraform will show: `["a", "b", "c"] => ["a", "b"]` but it will never propagate the change (as the API to update tags doesn't exist AFAIK). So, it will *always* show that change, until the Terraform code or the tags in the UI are manually updated to match.